README.md

```diff
@@ -1,2 +1,69 @@
-# windup-operator
-Windup Operator for Openshift-Kubernetes
+# windup-operator project
+
+This project creates an Operator to deploy Windup/MTA on an Openshift cluster ( Kubernetes one in future releases)
+
+It consists on a series of yaml files to deploy the operator (CRD, role, rolebinding, serviceaccount) and the native artifact compiled using Quarkus and GraalVM
+
+At this moment the operator reacts to creation of the Windup Custom Resource and will deploy all needed objects ( deployments, ingress, services, persistent volumes)
+
+## Building and pushing the Java code manually in native mode
+1. The following command will use the configuration in the `application.properties` file :  
+```
+quarkus.kubernetes.deployment-target=kubernetes, openshift
+quarkus.native.resources.includes=k8s/def/*.yaml
+quarkus.container-image.registry=quay.io
+quarkus.container-image.group=windupeng
+quarkus.container-image.name=windup-operator-native
+quarkus.container-image.tag=latest
+quarkus.kubernetes.service-type=load-balancer
+
+```
+1. Install JDK 11  
+You can use different ways , but SDKMan is very easy
+2. Execute the maven command:  
+`mvn clean package -Pnative -Dquarkus.native.container-build=true -Dquarkus.container-image.push=true`
+
+## Installation
+
+1. We can log in the Openshift cluster using `oc login .....` . You will need a user with cluster-wide permissions to deploy the CRD.
+1. Move to the `src/main/resources` folder
+1. If you are installing the Operator on a cluster without the `mta` namespace , you first should create the namespace with  
+  `oc apply -f windup.namespace.yaml`
+1. Create all the objects and deployment for the Operator. For convinience there's a file called `script.create.all.sh` that includes the execution of :  
```

Comment:
I executed the `script.create.all.sh` script but it failed because it was expecting the `mta` namespace for all the components but the CRD.
This is the log:
```
$ ./script.create.all.sh 
Creating serviceaccount
Error from server (NotFound): error when creating "./windup.serviceaccount.yaml": namespaces "mta" not found
Creating role
Error from server (NotFound): error when creating "./windup.role.yaml": namespaces "mta" not found
Creating rolebinding
Error from server (NotFound): error when creating "./windup.rolebinding.yaml": namespaces "mta" not found
Creating operator deployment
Error from server (NotFound): error when creating "./windup.deployment.yaml": namespaces "mta" not found
Creating CRD
customresourcedefinition.apiextensions.k8s.io/windups.windup.jboss.org created
```
So I feel like I misunderstood the namespace management a little bit.
Is it still mandatory to have the specific `mta` namespace in this installation scenario?

Reply:
This script is for manual installation purposes, and you need to specify the namespace. 
MTA or whatever you want , but you need to specify one. The same way that whenever you are installing the operator you need to specify it.
If you want to install everything in another namespace you need to create it before and change all these tests scripts.

Reply:
As above, I think it would be worth adding a short note that installation on a different namespace (not named `mta`) requires changes in all the scripts.